### PR TITLE
docs: rewrite README.md

### PR DESCRIPTION
**Pull request by danymat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,230 +2,207 @@
 
 <div align="center">
 
-<img src="res/neorg.svg" width=315>
+<img src="res/neorg.svg" width=300>
 
-# Neorg - An Organized Future
+# Neorg
```
Comment (mrossinek) [inline README.md:7]: Please revert this change.

File: README.md [hunk 2]
```diff
@@ -2,230 +2,207 @@
...
-[Installation](#wrench-installation)
+[Usage](#usage)
 •
-[Usage](#question-usage)
+[Modules](#modules)
+<br>
+[Roadmap](#roadmap)
+•
+[Philosophy](#philosophy)
+[FAQ](#faq)
```
Comment (mrossinek) [inline README.md:31]: Why is there no separating bullet between these?

File: README.md [hunk 3]
```diff
@@ -2,230 +2,207 @@
...
-[Keybinds](#keyboard-keybinds)
+[Credits](#credits)
 •
-[Wiki](#notebook-consult-the-wiki)
+[Support](#support)
 
-[GIFS](#camera-extra-gifs)
+</div>
 
-[Credits: Logo by Binx](#green_heart-credits)
```
Comment (mrossinek) [inline README.md:34]: This credit to Binx should not be removed.

Comment (mrossinek) [reply README.md:34]: I see, you created a `Credits` section. Maybe mention for Binx specifically the logo

File: README.md [hunk 4]
```diff
@@ -2,230 +2,207 @@
...
+## Summary
 
-</div>
+Neorg (_New_-_Organization_) is a tool designed to reimagine organization as you know it.
```
Comment (mrossinek) [inline README.md:43]: We should start with a catchy gif or screenshot of a fully fledged example file. Maybe similar to what [rich](https://github.com/willmcgugan/rich) does.

File: README.md [hunk 5]
```diff
@@ -2,230 +2,207 @@
...
-> The pain... it won't stop. After so much oppression from other text editors, it's time we fight back.
-With the introduction of lua, we *will* fight back.
+> Why do we need Neorg ?
 
-<div align="center">
+There are currently projects designed to [clone org-mode from emacs](https://github.com/kristijanhusak/orgmode.nvim),
+then what is the goal of this project?
 
----
+Whilst those projects are amazing, it's simply not enough for us. We need our _own, better_ solution - one that will
```
Comment (mrossinek) [inline README.md:52]:
```suggestion
Whil3 those projects are amazing, it's simply not enough for us. We need our _own and (more importantly) better_ solution - one that will
```

File: README.md [hunk 6]
```diff
@@ -2,230 +2,207 @@
...
+surpass _every_ other text editor.
 
-### Manage Your Life with Neovim-inspired Keybinds
-Keybinds that make logical sense. Simply think, don't remember.
+That's why we created _Neorg_.
 
-<img src="https://user-images.githubusercontent.com/76052559/132091379-845bf06d-7516-4c28-b32d-77b9734a44fe.gif">
+_IMPORTANT_: Neorg is _alpha_ software. We consider it stable however be prepared for changes and potentially outdated documentation. We are advancing fast and keeping docs up-to-date would be very painful.
```
Comment (mrossinek) [inline README.md:57]:
```suggestion
__IMPORTANT__: Neorg is _alpha_ software. We consider it stable however be prepared for changes and potentially outdated documentation. We are advancing fast and keeping docs up-to-date would be very painful.
```
Should be bold _imo_

File: README.md [hunk 7]
```diff
@@ -2,230 +2,207 @@
...
 
----
+To know more about the philosophy of Neorg, go [here](#philosophy).
 
-### Jump To The Most Important Directories with Workspaces
-Teleport to your favourite locations right away.
+## Showcase
 
+<details>
+<summary>A .norg file:</summary>
```
Comment (mrossinek) [inline README.md:64]:
```suggestion
<summary>A `.norg` file:</summary>
```

File: README.md [hunk 8]
```diff
@@ -2,230 +2,207 @@
...
+  <img width="700" alt="Capture d’écran 2021-12-29 à 10 50 59" src="https://user-images.githubusercontent.com/5306901/147649564-6f41e008-b292-4a67-84ab-82e73bc71890.png">
```
Comment (mrossinek) [inline README.md:65]: This screenshot should:

- have concealed icons
- concealed markup + links
- maybe be in zen mode? I.e. no line numbers, no line blame, no statusline, etc.

File: README.md [hunk 9]
```diff
@@ -2,230 +2,207 @@
...
+</details>
 
-</div>
+<details>
+<summary>Concealing module enabled:</summary>
+  <img width="700" alt="Capture d’écran 2021-12-29 à 10 53 26" src="https://user-images.githubusercontent.com/5306901/147649784-5654009e-c4aa-4f2b-8e30-571578a313b6.png">
```
Comment (mrossinek) [inline README.md:70]: This screenshot should use the `brave` concealing mode (i.e. use the unicode word joiner rather than actual whitespace). If the above comment applies, then it may become redundant.

File: README.md [hunk 10]
```diff
@@ -2,230 +2,207 @@
...
+</details>
 
-<div align="center">
-<img src="https://user-images.githubusercontent.com/13149513/125272567-cf8ba100-e32d-11eb-821d-f43f768570fb.gif">
-<img src="https://user-images.githubusercontent.com/13149513/125272579-d2869180-e32d-11eb-936b-1601086d0c73.gif">
-</div>
+<details>
+<summary>Manage your life with inspired keybinds:</summary>
```
Comment (mrossinek) [inline README.md:74]: Inspired by what? Should be `vim`-inspired I guess?

File: README.md [hunk 11]
```diff
@@ -2,230 +2,207 @@
...
+  
+  ![Enregistrement de l’écran 2021-12-29 à 11 03 57](https://user-images.githubusercontent.com/5306901/147650852-de6b2542-39c4-44c6-a228-ded867a71d4e.gif)
+</details>
 
----
+<details>
+<summary>Treesitter powered editing:</summary>
```
[the remark this answers is not in the document]
Comment (mrossinek) [reply README.md:80]: Maybe even `treesitter powered note-taking`?

File: README.md [hunk 12]
```diff
@@ -2,230 +2,207 @@
...
-<div align="center">
+  ![Enregistrement de l’écran 2021-12-29 à 11 19 01](https://user-images.githubusercontent.com/5306901/147652347-80daa0db-f5d5-46e7-b553-2922553c2c78.gif)
+</details>
 
-### Configure Everything - Literally
-Experience the power and configurability of Neorg's backend through modules and events.
-<br>
-Select only the code you want - throw everything else away.
+<details>
+<summary>Manage your tasks and project with GTD module:</summary>
```
[the remark this answers is not in the document]
Comment (mrossinek) [reply README.md:86]: Maybe also add a link to some GTD guide?

File: README.md [hunk 13]
```diff
@@ -2,230 +2,207 @@
...
+  And much more...
 
-</div>
+</details>
 
----
+<details>
+  <summary>Powerpoint-like presentations in neovim, with presenter module:</summary>
+  
+  ![Enregistrement_de_lecran_2021-11-15_a_22 40 36](https://user-images.githubusercontent.com/5306901/147654155-c2aa728a-5c2b-4813-b3e2-fbf7e2ffd2a2.gif)
```
Comment (mrossinek) [inline README.md:107]: This screencast contains some artifacts of the dimmed code block.

File: README.md [hunk 14]
```diff
@@ -2,230 +2,207 @@
...
 
-### TreeSitter Powered Editing
-Feel more accurate edits thanks to Neorg's deeper understanding of your documents with Treesitter
+  ![Enregistrement de l’écran 2021-12-29 à 11 47 59](https://user-images.githubusercontent.com/5306901/147657014-68573df8-0e43-4a8b-bb81-0db1c80cbbd8.gif)
+</details>
+</details>
 
-<img src="https://user-images.githubusercontent.com/76052559/132091729-6814a796-21a9-43af-a8f1-df7f44d1928b.gif">
+<details>
+  <summary>Get completion for various items in Neorg:</summary>
```
Comment (mrossinek) [inline README.md:118]: Which completion engines do we support? Should be listed in the details here.

File: README.md [hunk 15]
```diff
@@ -2,230 +2,207 @@
...
----
+<details>
+  <summary>Powerpoint-like presentations in neovim, with presenter module:</summary>
+  
+  ![Enregistrement_de_lecran_2021-11-15_a_22 40 36](https://user-images.githubusercontent.com/5306901/147654155-c2aa728a-5c2b-4813-b3e2-fbf7e2ffd2a2.gif)
+</details>
 
-<div align="center">
+<details>
+  <summary>Get syntax highlighting for any language supported by NeoVim:</summary>
```
Comment (mrossinek) [inline README.md:111]:
```suggestion
  <summary>Get syntax highlighting for any language supported by Neovim:</summary>
```

File: README.md [hunk 16]
```diff
@@ -2,230 +2,207 @@
...
-max amount of features that we can from the Neovim core, and maintaining the same version of a plugin
-for two different versions of Neovim is simply too much for the still rather small team that Neorg is comprised of.
-You can still use Neorg on `0.5.x`, however don't expect all modules to load properly.
+## Installation
 
-Installation may seem a bit daunting, however it's nothing you can't understand. If you really like to be in control,
-you can read exactly what the below code snippets do in the [wiki](https://github.com/nvim-neorg/neorg/wiki/Installation).
-You can install through any plugin manager (it can even be vimscript plugin managers, as long as you're running Neovim version 0.6 or higher).
+Neorg requires at least `0.6+` to operate.
+You can still use Neorg on `0.5.x`, however don't expect all modules to load properly.
```
Comment (mrossinek) [inline README.md:126]:
```suggestion
You can still use Neorg on `0.5.x`, however, do not expect all modules to load properly.
```

Comment (mrossinek) [reply README.md:126]: `load` or `work` properly?

File: README.md [hunk 17]
```diff
@@ -2,230 +2,207 @@
...
 
-> :exclamation: NOTE: Neorg requires [plenary.nvim](https://github.com/nvim-lua/plenary.nvim) to operate, so be sure to install it alongside Neorg!
+You can install through your favorite plugin manager:
```
Comment (mrossinek) [inline README.md:128]:
```suggestion
You can install Neorg with your favorite plugin manager. For example:
```

File: README.md [hunk 18]
```diff
@@ -2,230 +2,207 @@
...
-          }
-      },
-  }
-  ```
+- [Packer (with lazyloading)](https://github.com/wbthomason/packer.nvim):
 
-  Want to lazy load? Turns out that can be rather problematic. You can use the `ft` key to load Neorg only upon entering a .norg file.
-  Here's an example:
+  Want to lazy load? Turns out that can be rather problematic.
+  You can use the `ft` key to load Neorg only upon entering a .norg file:
```
Comment (mrossinek) [inline README.md:147]:
```suggestion
  You can use the `ft` key to load Neorg only upon entering a `.norg` file:
```

Comment (mrossinek) [reply README.md:147]: Similar things elsewhere.

File: README.md (hunk 16 above)
Comment (mrossinek) [reply README.md:126]: Also: similar abbrevations (as in `don't`) are use elsewhere.

File: README.md [hunk 19]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
-then try creating an `after/ftplugin/norg.lua` file and paste your Neorg configuration there.
+may not work right off the bat, however most commonly it's because of plugin loading order.
+
+Neorg needs `nvim-treesitter` to be up and running before it starts adding colours to highlight groups.
+
+Not using packer? Make sure that Neorg's `setup()` gets called after `nvim-treesitter`'s setup.
 
+If you're on Mac and have compilation errors when doing `:TSInstall`, check out this [fix](https://github.com/nvim-neorg/neorg/issues/74#issuecomment-906627223).
 It's a bit hacky - it will unfortunately stay this way until we get first-class support in the `nvim-treesitter` repository.
 Sorry!
```
Comment (mrossinek) [inline README.md:262]: https://github.com/nvim-treesitter/nvim-treesitter/pull/1795 is basically ready to be merged. I think we can hold off with merging these changes until that is the case.
This means, you could also integrate the changes done in #214 into this PR.

File: README.md [hunk 20]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
 
-### Setting up a Completion Engine
-Neorg comes with its own API for completion. Users can then write integration modules to allow different plugins like `nvim-compe` and `nvim-cmp`
-to communicate with the Neorg core. By default no engine is specified. To specify one, make sure to configure `core.norg.completion`:
+## Usage
+
+We recommend reading the [spec](docs/NFF-0.1-spec.md) and familiarizing yourself with the new format 🥳
```
Comment (mrossinek) [inline README.md:266]: Speaking of the spec, it also needs a rewrite...

File: README.md [hunk 21]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+
+You can even view a summary directly in your neovim instance, by doing `:h neorg`
+
+Simply drop into a `.norg` file and start typing!
+
+Or you can require the `core.norg.dirman` module, that'll help you manage workspaces:
```
Comment (mrossinek) [inline README.md:272]: This requires some more explanations:
- what are workspaces?
- what benefits do they bring, i.e. why do I need them?
- etc.

File: README.md [hunk 22]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+      }
+    }
+    ...
+  }
 }
 ```
 
-#### Cmp
-Make sure to enable the `neorg` completion source in the cmp sources table:
+For more information about what is a module, check out [here](https://github.com/nvim-neorg/neorg/wiki/Installation#the-concept-of-modules).
```
Comment (mrossinek) [inline README.md:329]:
```suggestion
For more information about what a module is, check out [this](https://github.com/nvim-neorg/neorg/wiki/Installation#the-concept-of-modules).
```

File: README.md [hunk 23]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+
+To know which configurations are provided by default for a module, just click on their link: you'll go to the module page in the [Wiki](https://github.com/nvim-neorg/neorg/wiki).
```
Comment (mrossinek) [inline README.md:331]: This seems a little detached from the previous. Where are those links of the modules? Etc.

File: README.md (hunk 22 above)
Comment (mrossinek) [reply README.md:329]: Also: is this link location still up-to-date?

File: README.md [hunk 24]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
 ```
 
-# :question: Usage
-Simply drop into a .norg file and start typing!
+This is a list of external modules:
+
+| Module name                                                                        | Description                                                                   |
+| :--------------------------------------------------------------------------------- | :---------------------------------------------------------------------------- |
+| [`utilities.gtd-project-tags`](https://github.com/esquires/neorg-gtd-project-tags) | Provides a view of tasks grouped with a project tag. Requires `core.gtd.base` |
+
```
Comment (mrossinek) [inline README.md:390]: Let's add a sentence suggesting plugin authors to open PRs and add links to their modules here!

File: README.md [hunk 25]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+We track a high-level roadmap, so that you can know what to expect. Just do `:h neorg-roadmap`.
+
+To know exactly what's being worked on, just check out the [PR's](https://github.com/nvim-neorg/neorg/pulls).
+
+## Philosophy
+
+We think of Neorg as a plugin that will give you all the bragging rights for using Neovim. Here's how we plan to do it:
+
+1. Revise the org format: we want it simple, very extensible, unambiguous. Will make you feel right at home. Org and markdown have several flaws, but the most
```
Comment (mrossinek) [inline README.md:401]: `Revise the org format` sounds like we are affiliated with it. This is not true. Instead, we are `building a file format for organization from the ground up`. Obviously we're taking inspiration from other great formats but we should stress our independence!

File: README.md [hunk 26]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+   notable one is the requirement for **complex parsers**.
+   I really advise checking some writeups out on how bad it can get at times.
```
Comment (mrossinek) [inline README.md:403]: First person singular is not used anywhere else and should not be used here.

File: README.md [hunk 27]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+   What if we told you it's possible to alleviate those problems, all whilst keeping that familiar feel?
+   Enter the .norg file format, whose [base spec](docs/NFF-0.1-spec.md) is practically complete.
+   The cross between all the best things from org and the best things from markdown, revised and merged into one.
+
+2. Keybinds that _make sense_: vim's keybind philosophy is unlike any other, and we want to keep that vibe.
```
Comment (mrossinek) [inline README.md:408]: We're stressing this but have not revised the keybinds yet. This should become high priority after (or even before) this PR!

File: README.md [hunk 28]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
-We tried focusing heavily on the backend first, but now that that is almost done we are actually starting work on features just for you:
-- [x] Telescope.nvim integration for several things (see https://github.com/nvim-neorg/neorg-telescope)
-- [x] TreeSitter parser (can be found [here](https://github.com/nvim-neorg/tree-sitter-norg))
-    - [x] AST Generation
-    - [x] Custom highlight support
-    - [x] Custom folds
-    - [x] Language injection (for code blocks)
-    - [x] Smarter todo item toggling with the TreeSitter AST
+The wiki is the go-to place if you need answers to anything Neorg-related. Usage, Keybinds, User Callbacks, Modules, Events?
+It's all there, so we recommend you seriously go [read it](https://github.com/nvim-neorg/neorg/wiki)!
```
Comment (mrossinek) [inline README.md:421]: Let's also point to our Discord again.

File: README.md [hunk 29]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
+
+## Credits
+
+Massive shoutouts go to the people who supported the project, and help out in creating a good user experience! These are:
+
+- [vhyrro](https://github.com/vhyrro)
+- [mrossinek](https://github.com/mrossinek)
+- [danymat](https://github.com/danymat)
+- [Binx](https://github.com/Binx-Codes/)
+- [bandithedoge](https://github.com/bandithedoge)
```
Comment (mrossinek) [inline README.md:439]: I have seen in other READMEs that the profile pics are also included in a mosaic like raster. Can we do smth similar?

File: README.md [hunk 30]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
-Here are some things we *are* working on:
-- Fully fledged GTD workflow (with @Danymat)
-- Dynamically displaying and interacting with a Table of Contents (with @mrossinek)
-- Better parsing of markup (bold, italic etc.)
-- Overhauled indentation engine
+> Love what I do? Want to see more get done faster? Want to support future projects of mine? Any sort of support is always
+> heartwarming and fuels the urge to keep going :heart:. You can support me here:
 
+- [Buy me a coffee!](https://buymeacoffee.com/vhyrro)
+- [Support on LiberaPay](https://liberapay.com/vhyrro)
```
Comment (mrossinek) [inline README.md:449]:
```suggestion
- [Support me on LiberaPay](https://liberapay.com/vhyrro)
```

File: README.md (hunk 3 above)
Comment (max397574) [reply README.md:34]: > I see, you created a `Credits` section. Maybe mention for Binx specifically the logo
Yes I like that idea

**Comment by roland-rollo** [comment]
Dears, what do you think about changing Neovim button background color to other that actually is now? Maybe it's my eyes problem, but I barely see N that is Neovim logo - it's blend to background.

**Comment by danymat** [comment]
> Dears, what do you think about changing Neovim button background color to other that actually is now? Maybe it's my eyes problem, but I barely see N that is Neovim logo - it's blend to background.

Hello @roland-rollo , can you check again ? I removed the Neovim logo because it was a pain to have a nice color behind it

**Comment by roland-rollo** [comment]
@danymat Looks good. Thank you!

**Comment by danymat** [comment]
This PR is on hold on demand of @vhyrro

File: README.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (vuki656) [reply README.md:78]: Could just be `treesitter powered`?

File: README.md [hunk 31]
```diff
@@ -269,184 +263,214 @@ require('nvim-treesitter.configs').setup {
...
+Changing workspaces is easy, just do `:Neorg workspace work`, where `work` is the name of your workspace.
+Note that `:Neorg` is only available when the Neorg environment is loaded, i.e. when you're
+in a .norg file or have loaded a .norg file already in your Neovim session.
+
+If the Neorg environment isn't loaded you'll find a `:NeorgStart` command which will launch Neorg and pop
+you in to your last (or only) workspace.
+
+> It works, cool! What are the next steps?
+
+We recommend you add some core modules that can greatly improve your experience, such as:
```
Comment (mrossinek) [inline README.md:313]:
```suggestion
We recommend you add some core modules (continue reading to find out how) that can greatly improve your experience, such as:
```

File: README.md [hunk 32]
```diff
@@ -269,184 +263,214 @@ require('nvim-treesitter.configs').setup {
...
+- Setting up a completion engine (`core.norg.completion`)
+- Using the concealer module to enable icons (`core.norg.concealer`)
+
+## Modules
+
+As you surely saw previously, we loaded `core.defaults`, and recommended that you load `core.norg.dirman`.
+Sooo, what are they exactly? We'll give you a brief explanation.
+
+Modules are basically isolated bits of code that provide a specific subset of features. They can be docked in
+to the environment at any time and can be essentially stacked together like lego bricks!
```
Comment (mrossinek) [inline README.md:324]:
```suggestion
Modules are basically isolated bits of code that provide a specific subset of features. They can be docked into
the environment at any time and can be essentially stacked together like lego bricks!
```

File: README.md [hunk 33]
```diff
@@ -269,184 +263,214 @@ require('nvim-treesitter.configs').setup {
...
+| [`core.presenter`](https://github.com/nvim-neorg/neorg/wiki/Core-Presenter)     | Neorg module to create gorgeous presentation slides.                        |
+| [`core.norg.completion`](https://github.com/nvim-neorg/neorg/wiki/Completion)   | A wrapper to interface with several different completion engines.           |
+| [`core.norg.concealer`](https://github.com/nvim-neorg/neorg/wiki/Concealer)     | Enhances the basic Neorg experience by using icons instead of text.         |
+| [`core.norg.journal`](https://github.com/nvim-neorg/neorg/wiki/Journal)         | Easily create files for a journal.                                          |
+| [`core.gtd.base`](https://github.com/nvim-neorg/neorg/wiki/Getting-Things-Done) | Manages your tasks with Neorg using the Getting Things Done methodology.    |
+| [`core.norg.dirman`](https://github.com/nvim-neorg/neorg/wiki/Dirman)           | This module is be responsible for managing directories full of .norg files. |
+| [`core.norg.qol.toc`](https://github.com/nvim-neorg/neorg/wiki/Qol-Toc)         | Generates a Table of Contents from the Neorg file.                          |
```
Comment (mrossinek) [inline README.md:367]: Can we sort this list alphabetically?

File: README.md [hunk 34]
```diff
@@ -269,184 +263,214 @@ require('nvim-treesitter.configs').setup {
...
+```
+
+<details>
+<summary>List of community modules:</summary>
+
+| Module name                                                                        | Description                                                                          |
+| :--------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------- |
+| [`external.gtd-project-tags`](https://github.com/esquires/neorg-gtd-project-tags) | Provides a view of tasks grouped with a project tag. Requires `core.gtd.base`        |
+| [`external.integrations.gtd-things`](https://github.com/danymat/neorg-gtd-things) | Use Things3 database to fetch and update tasks instead. Requires `core.gtd.base`        |
+| [`core.integrations.telescope`](https://github.com/nvim-neorg/neorg-telescope)     | Neorg integration with [Telescope](https://github.com/nvim-telescope/telescope.nvim) |
```
Comment (mrossinek) [inline README.md:402]: Why is there a `core` module here?

File: README.md (hunk 26 above)
Comment (mrossinek) [reply README.md:416]: Also: do we have some suggested writeups?

File: README.md [hunk 35]
```diff
@@ -269,184 +263,214 @@ require('nvim-treesitter.configs').setup {
...
+if you care about what makes Neorg tick.
+
+## Philosophy
+Our goals are fairly simple:
+
+1. Revise the org format: simple, extensible, unambiguous. Will make you feel right at home. Alternate markup formats have several flaws, but the most
+   notable one is the requirement for **complex parsers**.
+   I really advise checking some writeups out on how bad it can get at times.
+   What if we told you it's possible to alleviate those problems, all whilst keeping that familiar feel?
+   Enter the .norg file format, whose [base spec](docs/NFF-0.1-spec.md) is practically complete.
```
Comment (mrossinek) [inline README.md:418]:
```suggestion
   Enter the `.norg` file format, whose [base spec](docs/NFF-0.1-spec.md) is practically complete.
```

File: README.md [hunk 36]
```diff
@@ -269,184 +263,214 @@ require('nvim-treesitter.configs').setup {
...
+We track a high-level roadmap, so that you can know what to expect. Just do `:h neorg-roadmap`.
+To know exactly what's being worked on, just check out the [repo's PRs](https://github.com/nvim-neorg/neorg/pulls).
+
+## FAQ
+
+<!-- TODO(vhyrro): Populate with common issues -->
 
-# :notebook: Consult The Wiki
 The wiki is the go-to place if you need answers to anything Neorg-related. Usage, Keybinds, User Callbacks, Modules, Events?
 It's all there, so we recommend you seriously go [read it](https://github.com/nvim-neorg/neorg/wiki)!
```
Comment (mrossinek) [inline README.md:441]: We should also point to Discord here

**Review by mrossinek — COMMENTED** [review]
Some more comments. The GIFs still need to be updated, too

File: README.md [hunk 37]
```diff
@@ -2,231 +2,220 @@
...
+<details>
+  <summary>Get completion for various items in Neorg:</summary>
+
+  ![Get completion for various items in Neorg](https://user-images.githubusercontent.com/5306901/147657095-aa51a609-5bc2-4aa4-9687-bdda6ef48860.gif)
+</details>
+
+## Installation
+
+Neorg requires at least Neovim 0.6+ to operate.
+You can still use Neorg on `0.5.x`, however don't expect all modules to load properly.
```
[the remark this answers is not in the document]
Comment (danymat) [reply README.md:126]: Hmm the tests passes on 0.6.0 and nightly. We can bump the requirements in the readme

File: README.md [hunk 38]
```diff
@@ -2,231 +2,220 @@
...
+require('neorg').setup {
+    load = {
+        ["core.defaults"] = {}
+    }
+}
+```
+
+You can see [here](https://github.com/nvim-neorg/neorg/wiki#default-modules) which modules are automatically required when adding `core.defaults`.
+
+### Treesitter
```
[the remark this answers is not in the document]
Comment (danymat) [reply README.md:213]: Treesitter is optional for neorg to work, but it's too much required on neorg core modules. I guess we should note this to the users

File: README.md (hunk 34 above)
Comment (danymat) [reply README.md:402]: Because it's not renamed iirc. I'm afraid to rename it (because it'll stop working for users), but I think we should do it

File: README.md [hunk 39]
```diff
@@ -269,102 +249,210 @@ require('nvim-treesitter.configs').setup {
...
 
 **Still not working**? Uh oh, you're stepping on muddy territory. There are several reasons why a parser
-may not work right off the bat, however most commonly it's because of plugin loading order. Neorg needs
-`nvim-treesitter` to be up and running before it starts adding colours to highlight groups.
-With packer this can be achieved with an `after = "nvim-treesitter"` flag in your `use` call to Neorg.
-Not using packer? Make sure that Neorg's `setup()` gets called after `nvim-treesitter`'s setup. If nothing else works
-then try creating an `after/ftplugin/norg.lua` file and paste your Neorg configuration there.
+may not work right off the bat, however most commonly it's because of plugin loading order.
+
+Neorg needs `nvim-treesitter` to be up and running before it starts adding colours to highlight groups.
```
[the remark this answers is not in the document]
Comment (mrossinek) [reply README.md:270]: Resolving this. A link is now included shortly above.

File: README.md (hunk 34 above)
Comment (vhyrro) [reply README.md:402]: Initially it was called `core.integrations.telescope` because it was made by our core team. Now that I think about it tho it doesn't really make sense to have it named like that lol

File: README.md (hunk 38 above)
[the remark this answers is not in the document]
Comment (vhyrro) [reply README.md:185]: I've moved this to the installation section - you will get an incredibly mediocre experience if you don't set up treesitter, simply too many modules want it in order for you to get any serious features.

File: README.md (hunk 37 above)
[the remark this answers is not in the document]
Comment (vhyrro) [reply README.md]: 0.6 is the standard now, so might as well require users to be up to date. Will be less of a hassle to deal with :)

File: README.md (hunk 36 above)
Comment (vhyrro) [reply README.md:440]: I decided to not do this in the end cause Contributing (the next section) also has a link to the discord in it. It feels a little aggressive cause they're both close together.
